clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/utils/HadoopUtils.java

```diff
@@ -17,6 +19,9 @@
  */
 public class HadoopUtils {
   private static final Logger LOGGER = LogManager.getLogger(HadoopUtils.class);
+  private static final byte SEQUENCE_FILE_VERSION_WITH_METADATA = (byte) 6;
```

Comment:
In my opinion, this isn’t necessary. We can simply use the sequence file reader, and if it throws an IOException indicating it’s not a SequenceFile, we can rely on that exception to determine the file type.

Reply:
I don't want to rely on arbitrary things that are not specified as a contract. The file header is a standard contract and I would trust that more than the message of an exception

Reply:
Why is that arbitrary?

Reply:
It’s just as arbitrary as relying on the file header. There’s a reason the file header wasn’t made public, so I don’t think this is a reasonable argument.

Reply:
The file header is a standard contract - Every file type uses a dedicated header, and that is how every system detects the type of the file. For example Avro uses 'Obj1' as the file header. Let me find the documentation for SequenceFile

Reply:
https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/io/SequenceFile.html - The section on `SequenceFile Formats` documents mentions what constitues a `SequenceFile`